nni/compression/pytorch/quantization_speedup/common.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import pycuda.driver as cuda
+import pycuda.autoinit
+import tensorrt as trt
+
+pycuda.autoinit
+
+EXPLICIT_BATCH = 1
+
+def GiB(val):
+    return val * 1 << 30
+
+# Simple helper data class that's a little nicer to use than a 2-tuple.
+class HostDeviceMem(object):
+    def __init__(self, host_mem, device_mem):
+        self.host = host_mem
+        self.device = device_mem
+
+    def __str__(self):
+        return "Host:\n" + str(self.host) + "\nDevice:\n" + str(self.device)
+
+    def __repr__(self):
+        return self.__str__()
+
+# Allocates all buffers required for an engine, i.e. host/device inputs/outputs.
+def allocate_buffers(engine):
```

Comment:
it is a little strange to put these tensorrt specific functions to `common.py`

Reply:
All functions in it are about cuda memory operation. I think it is better to take them out to make code easier understand.

Reply:
i agree, so you can rename this file, for example, "trt_cuda.py". because we supposed to support different backends, cuda memory operations are still specific for nvidia gpu.

Reply:
Have mv `common.py` to 'trt_pycuda.py'.